[MS-768] Social bar

stochna opened the pull request.

protyagov commented on src/part/social-bar.tsx:
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Locale } from "src/i18n/locale";
```
I think we do not need to import Locale here since it is not used

stochna replied:
@protyagov oh, I forgot that I wanted to ask, probably we should add alt for links?

protyagov replied:
Right, would be good to add ALT links -- they help Google when they index things.

stochna replied:
I meant title, not alt

protyagov replied:
Right, something like this:
`alt={_("FOOTER.APPLE_BUTTON.ALT")} `

protyagov reviewed (approved):
All good, we could remove Locale from line 2 in social-bar since it is not used there.

stochna commented:
@protyagov I used titles instead of alt as alt is not allowed on anchor links. Also found and fixed one missing localization and renamed APPLE_BUTTON to APPLE_BTN as we always use "BTN" naming.
